contracts/ModuleFactoryBase.sol

```diff
@@ -47,17 +47,18 @@ abstract contract ModuleFactoryBase is
         public
         view
         virtual
-        override(ERC165, IModuleFactory, AccessControl)
+        override(AccessControl, ERC165Storage)
         returns (bool)
     {
         return
-            interfaceId == type(IModuleFactory).interfaceId ||
+            interfaceId == type(IModuleFactoryBase).interfaceId ||
             super.supportsInterface(interfaceId);
     }
```

Comment:
Wait, you can straight up remove this whole function from here, right?

Reply:
So ModuleFactoryBase inherits from OZ's AccessControl, which inherits from ERC165, so this override is needed since this contract inherits from two different contracts with `supportsInterface`

Reply:
oh. in that case, is `ERC165Storage` even needed in this contract? Is it even _used_? What happens when a client calls `supportsInterface()` on this contract? Does that call chain ever even make it to the abstract `ERC165Storage` contract? 

I guess what I'm asking is, when multiple contracts are specified in the `override()` decorator... what's going on there?

Reply:
Yeah, so the multiple contracts listed in the override() decorator just specify which contracts contain that overridden function. So when `super` is called, it uses the implementation of whichever contract is highest up in the inheritance.

But yes you're right, the ERC165Storage is redundant and not needed here, so I removed in the most recent commit.

Reply:
next question (sorry to keep dragging this conversation on):

does that decision to remove `ERC165Storage` from `ModuleFactoryBase` affect the `supportsInterface` patterns in the actual module factory contracts that inherit from `ModuleFactoryBase`?

Might be weird if a module developer needs to use different ERC165 patterns in their module contracts vs their factory contract.

Reply:
Ok, so as we talked about, in the latest commit:
 - Made ModuleFactoryBase inherit from Ownable instead of AccessControl to prevent ERC165 function collision
 - Remove the supportsInterface function since it is no longer needed

Reply:
LET'S FUCKING GOO